mslib/mswms/wms.py

```diff
@@ -544,7 +544,8 @@ def produce_plot(self, query, mode):
                     logging.error("ERROR: %s %s", type(ex), ex)
                     msg = "The data corresponding to your request is not available. Please check the " \
                           "times and/or path you have specified.\n\n" \
-                          f"Error message: {ex}"
+                          f"Error message: {ex}.\n" \
+                          f"Hint: Check used waypoints."
```

Comment:
Is the f'string notation needed in this line?

Reply:
No, we not need it.
Previously I am not aware about use of f'string, so used it by mistake.